public/retro-fonts.css

```
@@ -1,26 +1,25 @@
-  src:url("/fonts/monaco.woff") format("woff");
-  src:url("/fonts/monaco.woff2") format("woff2")
+  font-family: Monaco;
+  src: url("/fonts/monaco.woff") format("woff");
+  src: url("/fonts/monaco.woff2") format("woff2");
 }
 
 @font-face {
-  font-family:Chicago_12;
-  src:url("/fonts/ChiKareGo2.woff") format("woff");
-  src:url("/fonts/ChiKareGo2.woff2") format("woff2")
+  font-family: Chicago_12;
```

Stylelint (17.12.0)

[error] Expected quotes around "Chicago_12" (font-family-name-quotes)

(font-family-name-quotes)